var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -3,20 +3,23 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+import spack
 
 
-class PyBglibpy(PythonPackage):
+class PyBglibpy(spack.PythonPackage):
     """Pythonic Blue Brain simulator access"""
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/sim/BGLibPy"
-    git      = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
+    git = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
-    version('develop', branch='master')
-    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545', preferred=True)
+    spack.version('develop', branch='master')
+    spack.version(
+        '4.0.27',
+        commit='42d9c1f891ef1ec9af6d72c49ff3b7726a009951',
+        preferred=True)
```

Comment:
Along with new as preferred, we prefer to keep few old version so that one can easily build older version.

Reply:
IMO, `preferred` should only be set if an old version is preferred, and otherwise omitted. Makes version updates simpler, and spack automatically picks the newest version.

Reply:
Ok, I didn't add this preferred=True option myself, but I've removed it